W-18292259-aggregators-module-clustering-kt (#2887)
mulesoft/docs-connectors: aggregators/1.0/modules/ROOT/pages/aggregators-cluster.adoc

= Using Aggregators in a Cluster

[IMPORTANT]

> Aggregators don’t work with multiple workers and a persistent object store, because clustering is not available in CloudHub.

Reviewer: ```suggestion
Aggregators don’t work with multiple workers and a persistent object store because clustering isn't available in CloudHub.
```
